### PR TITLE
Fix/15516198 custom extensions data

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

File: src/components/shared/Extensions/Accordion/partials/AccordionNested/AccordionNestedItem.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            initial="hidden"
+            animate="visible"
+            exit="hidden"
+            variants={accordionAnimation}
+            transition={{ duration: 0.4, ease: 'easeOut' }}
+            className="overflow-hidden"
+          >
+            <div
+              className="mt-1 text-sm text-gray-600"
+              dangerouslySetInnerHTML={{ __html: contentText }}
```
Comment (CzarekDryl) [inline src/components/shared/Extensions/Accordion/partials/AccordionNested/AccordionNestedItem.tsx:47]: Can't this be just inside div? => `<div>{contentText}</div>`

File: src/components/shared/Extensions/Accordion/partials/AccordionHeader.tsx [hunk 2]
```diff
@@ -2,33 +2,44 @@ import React, { FC } from 'react';
...
-          className={clsx({
+    <button
+      onClick={onClick}
+      onKeyUp={onKeyUp}
+      type="button"
+      className="flex justify-between items-center py-2 w-full group border-b border-gray-200 text-1"
+      aria-label={formatMessage({
+        id: isOpen ? 'ariaLabel.closeAccordion' : 'ariaLabel.openAccordion',
+      })}
+    >
```
[the remark this answers is not in the document]
Comment (KatarzynaStefanowiczPagepro) [reply src/components/shared/Extensions/Accordion/partials/AccordionHeader.tsx:30]: Done

**Review by CzarekDryl — APPROVED** [review]

File: src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx [hunk 3]
```diff
@@ -261,6 +368,8 @@ export const useLazyConsensusPage = (
     }
   };
 
+  // const initSelectFields = useCallback(() => {}, []);
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx:371]: please remove commented code

File: src/components/shared/Extensions/Accordion/Accordion.tsx [hunk 4]
```diff
@@ -24,7 +24,7 @@ const Accordion: FC<AccordionProps> = ({
   };
 
   return (
-    <div className="bg-base-white">
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/Accordion/Accordion.tsx:27]: are you sure about this change?

Comment (KatarzynaStefanowiczPagepro) [reply src/components/shared/Extensions/Accordion/Accordion.tsx:27]: As I can see this component is used only in LazyConsensusPage and works correctly

File: src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx (hunk 3 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/frame/Extensions/pages/LazyConsensusPage/hooks.tsx]: Removed

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**